cmd/bucket-replication-utils.go

```diff
@@ -638,7 +638,7 @@ type replicationResyncer struct {
 }
 
 const (
-	replicationDir      = "replication"
```

Comment:
better to make this change as a separate PR - both mrf and stats use this directory. It needs to be backwards compatible to move the stats and mrf files to new path